### PR TITLE
fixed a few minor mobile issues

### PR DESCRIPTION
**Pull request by bthaile**
There were some mobile market creation styling issues:

Screen Shot 2019-11-07 at 11 06 41 AM

Screen Shot 2019-11-07 at 11 08 56 AM

User could not create orders on mobile in market creation form.
IMG_4151

Screen Shot 2019-11-07 at 11 15 46 AM

hide advanced button for liquidity
image

align max label and remove extra space
image

fixed button jumping
image

### CODE REVIEW

**Comment by phoebemirman** [comment]
We need to hide the advanced button here.  I think andys pr has a fix for that

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🍕

 This link will always point to this PR's latest changes: https://peek.run/w9jhkpe

**Review by phoebemirman — CHANGES_REQUESTED** [review]
can you make it so add order button doesnt jump around

Screen Shot 2019-11-07 at 9 41 42 AM
Screen Shot 2019-11-07 at 9 41 36 AM

also advanced button shouldn't be on order form

**Review by phoebemirman — CHANGES_REQUESTED** [review]
Screen Shot 2019-11-07 at 9 56 00 AM


can you decrease the top margin on max cost. looks off

**Review by phoebemirman — CHANGES_REQUESTED** [review]
where'd the svg go?
Screen Shot 2019-11-07 at 10 58 26 AM

File: packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less [hunk 1]
```diff
@@ -24,6 +24,10 @@
     &.Full {
       flex-grow: unset;
     }
+
+    &.Liquidity {
+      flex-grow: inherit;
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less]: you can just say 1, don't need to put inherirt

Comment (bthaile) [reply packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less]: unset doesn't work

Comment (phoebemirman) [reply packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less]: flex-grow: 1 should work

Comment (bthaile) [reply packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less]: removed it.

**Comment by bthaile** [comment]
made changes. 

image

File: packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less [hunk 2]
```diff
@@ -24,10 +24,6 @@
     &.Full {
       flex-grow: unset;
     }
-
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less]: i think you did need this, it jumps around again

**Review by phoebemirman — CHANGES_REQUESTED** [review]
you need flex-grow: 1

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**